modules/ROOT/pages/modules/jmx.adoc

```diff
@@ -13,10 +13,9 @@ from Maven Central: http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22org.apache
 * The Aries Util bundle, e.g.
 from Maven Central: http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22org.apache.aries.util%22[org.apache.aries.util]
 
-As prescribed by the OSGi JMX specification, to hook it in with an MBean Server the relevant MBean Server needs to be registered in the OSGi Service Registry.
+As prescribed by the OSGi JMX specification, to hook it in with an MBean Server the relevant MBean Server needs to be registered in the OSGi Service Registry. A simple bundle that registers the Platform MBean Server is available on Maven Central: https://search.maven.org/search?q=a:org.apache.aries.jmx.mbeanserver-platform[org.apache.aries.jmx.mbeanserver-platform].
```

Comment:
I would wait a day or two until the new artifact appear on https://mvnrepository.com/artifact/org.apache.aries.jmx/org.apache.aries.jmx.mbeanserver-platform and use this link.

Reply:
Page  is now up, force-pushed the change.